CPP17.md

```diff
@@ -423,15 +423,15 @@ class Proxy {
 public:
   Proxy(Callable c): c(c) {}
   template <class... Args>
-  decltype(auto) operator()(Args&&... args) {
+  auto operator()(Args&&... args) {
```

Comment:
I don't think this one should change. A true proxy class (one that just acts as a wrapper for calling a callable such as in this example), should have `decltype(auto)` because it will also account for if the callable returns a reference type -- whereas with `auto`, it will return a value type.

Reply:
makes sense. will do.